warp/src/raygun/mod.rs

```diff
@@ -525,7 +525,7 @@ impl Conversation {
     }
 }
 
-#[derive(Debug, Hash, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Display)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Display)]
 #[serde(rename_all = "lowercase")]
 #[repr(C)]
 pub enum ConversationSettings {
```

Comment:
As we do add permissions in, we could probably go on and get rid of `ConversationSettings`

Reply:
ConversationSettings removed 👍